packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -125,7 +127,10 @@ class TwinDeploymentHandler {
       }
       await new Promise(f => setTimeout(f, 2000));
     }
-    throw new TimeoutError(`Deployment with contract_id: ${contract_id} failed to be ready after ${timeout} minutes.`);
+
+    const timeoutMessage = timeout > 1 ? `${timeout} minutes` : `${timeoutInSeconds} seconds`;
+
+    throw new TimeoutError(`Deployment with contract_id: ${contract_id} failed to be ready after ${timeoutMessage}.`);
```

Comment:
<img width="618" alt="image" src="https://github.com/user-attachments/assets/aa031223-76df-4497-847d-d941238c9b7d" />
This works fine if the time is less than or equal to one minute, but what if the time is equal to 1.5 minutes I think we will have the same issue

Reply:
Do you have any suggestions?